minor clarifications to contributing rock-ons doc section. Fixes #238 (#239)
rockstor/rockstor-doc: contribute_rockons.rst

`github.com <https://github.com/rockstor/rockon-registry>`_.
As detailed in the :ref:`adding_rockons` section,
please refer to the rockon-registry `README.md <https://github.com/rockstor/rockon-registry/blob/master/README.md>`_
for full instructions and examples on how to write your own Rock-on definition file.

If you are interested in contributing a Rock-on to enable your chosen project on Rockstor,
please submit a pull-request to the `rockon-registry <https://github.com/rockstor/rockon-registry>`_.
Steps for doing so are similar to those required for :ref:`contributetorockstor` or :ref:`contributedocs`.
If you are familiar with git then working on an issue branch in a local clone of your GitHub fork is favoured.
If you are not then the following is a quick and easy method but far less flexible.

A GitHub account is required.

1. Create a *New issue* (button) in the **rockon-registry** `issues section <https://github.com/rockstor/rockon-registry/issues>`_ and explain what you are proposing.
2. Creating your own *Fork* (button) of the `rockon-registry <https://github.com/rockstor/rockon-registry>`_ (GitHub login required).

> 3. Write and test your Rock-on definition file following the `README.md <https://github.com/rockstor/rockon-registry/blob/master/README.md>`_.

Reviewer: Refer to the file as "JSON definition file" for consistency with point 4 below.
```suggestion
3. Write and test your Rock-on JSON definition file following the `README.md <https://github.com/rockstor/rockon-registry/blob/master/README.md>`_.
```

  user intervention at the command line before and/or after.
* Finally, the image size is another helpful factor, with smaller images being 
  usually favored.

.. _deleterockons:

Deleting a Rock-on
==================

> The Rock-ons repository is predominantly community maintained / lead.

Reviewer: ```suggestion
The Rock-ons repository is predominantly community maintained/led.
```

> As such we depend on community involvement to maintain it's health.

Reviewer: ```suggestion
As such we depend on community involvement to maintain its health.
```

> On occasions a Rock-on will fall into dis-repair.

Reviewer: ```suggestion
On occasions a Rock-on will fall into disrepair.
```

> If you find such a Rock-on, ie broken or build on an abandoned/deprecated docker image,

Reviewer: ```suggestion
If you find such a Rock-on, _i.e._ broken or built on an abandoned/deprecated docker image,
```

then please report this on our `friendly forum <https://forum.rockstor.com/>`_.

> If there is then no community will/effort to maintain/repair that rock-on then we will happily accept a pull request to delete it.

Reviewer: ```suggestion
If there is then no community will/effort to maintain/repair that rock-on, then we will happily accept a pull request to delete it.
```

> Such a pull request would include the removal/deletion of the associated JSON definition file and it's associated entry within the root.json file.

Reviewer: ```suggestion
Such a pull request would include the removal/deletion of the associated JSON definition file and its associated entry within the root.json file.
```
